code/game/machinery/computer/communications.dm

```diff
@@ -203,7 +203,12 @@ var/shuttle_call/shuttle_calls[0]
 			var/response = alert(usr,"Are you sure you want to request a response team?", "ERT Request", "Yes", "No")
 			if(response != "Yes")
 				return
-			trigger_armed_response_team(1)
+			var/ert_reason = stripped_input(usr, "Please input the reason for calling an Emergency Response Team. This may be all the information they get before arriving at the station.", "Response Team Justification")
+			if(!ert_reason || !Adjacent(usr) || usr.restrained() || usr.isUnconscious())
```

Comment:
Replace the last two conditions with just `usr.incapacitated()`, it will cover all the cases.